docs/releases/release_checklist.md

```diff
@@ -5,42 +5,58 @@ This is a checklist for cutting a release
+    * Compare the dependency versions from the output of the pip-compile command to the current dependency versions listed in the `requirements.txt` file. Upgrade `requirements.txt` if necessary.
     * Run `pip install .` to install tern.
     * Run appropriate tests. Roll back requirements if necessary.
     * When satisfied, run `pip-compile --generate-hashes --output-file docs/releases/v<release>-requirements.txt`.
 
 - [ ] Write release notes.
-    * Summary
-    * New Features (if any)
-    * Deprecated Features (if any)
-    * Bug Fixes (if any)
-    * Resolved Technical Debt (if any)
-    * Future Work
-    * Changelog
+    * Create a new file for the release notes: `docs/releases/v<release>.md`
+    * If you are writing release notes for a patched release, only include:
+      - A link to the primary release notes.
+      - A brief summary of what the patched release changes do.
+      - A list of patches since the last release was cut. You can get this information by running `git log --oneline` and finding the commits since the tag.
+
+    * For any other release, include the following in your notes:
+      - Summary
+      - New Features (if any)
+      - Deprecated Features (if any)
+      - Bug Fixes (if any)
+      - Resolved Technical Debt (if any)
+      - Future Work
+      - Changelog     
         * "Note: This changelog will not include these release notes"
         * "Changelog produced by command: `git log --pretty=format:"%h %s" v<tag>..master`"
-    * Contributors (look at Authors in the changelog `git log --pretty=format:"%an %ae" v<tag>..master | uniq`)
-    * Contact the Maintainers
+      - Contributors (look at Authors in the changelog `git log --pretty=format:"%an %ae" v<tag>..master | uniq`)
+      - Contact the Maintainers
 
-- [ ] Commit release notes, `v<release>-requirements.txt`, and any changes to `requirements.txt`.
+- [ ] Commit release notes and create patch for your changes
+    * `git add` and `git commit` any changes. This will likely include`v<release>-requirements.txt`, any changes to `requirements.txt` and `v<release>.md`. **Do not push these changes to master!**
+    * Run `git format-patch -n1`. This will create a patch file of the release changes you just committed called `0001-<commit_title>.patch`.
+    * Open a new terminal and `cd` into a development virtual environment that contains your forked version of the Tern repo. `cd` into the forked Tern repo directory.
+    * Create a new branch. You will use this branch to submit a PR for the release changes.
+    * Copy the patch file you just created into your new forked repo environment.
+    * Run `git am 0001-<commit_message_title>.patch`.
+    * After running the `git am` command above, the changes in the patch will be available in your forked repo. You can verify this by running `git log` and looking at the top commit from the output.
+    * Open a pull request in the Tern project repository for your release changes.
```

Comment:
Can you add another bullet with something like "Request a review from another maintainer. Update as needed. Merge the PR. This commit is where the release will be tagged."

Reply:
Sure thing!